cabal-install/Main.hs

```diff
@@ -258,6 +256,7 @@ mainWorker args = topHandler $
       , regularCmd runCommand runAction
       , regularCmd initCommand initAction
       , regularCmd configureExCommand configureAction
+      , regularCmd reconfigureCommand reconfigureAction
```

Comment:
@23Skidoo Do you think there is some documentation I could add to clarify? The command is something I've often wanted; if it's hidden, I don't really see what the point of having it is at all.